### DISCUSSION TITLE
144

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#80] [General]
# 潮流周刊第144期 - 街舞少女

封面图来源于周末在天街吃饭后，看到楼下正在举办的 POPPING 街舞大赛，很有节奏感，站在那儿看了好久，年轻真好啊。

https://weekly.tw93.fun/posts/144-%E8%A1%97%E8%88%9E%E5%B0%91%E5%A5%B3

**Comment by robbietree8** [comment]
“面向教师和学生的实用人工智能课程”的链接有误

**Reply by tw93** [reply]
谢谢反馈，已经修改

**Comment by somkanel** [comment]
汤老师，你也是年轻的！

**Reply by tw93** [reply]
哈哈哈 你也是的

**Comment by siyi98** [comment]
答应我，以后周五下班了去酒吧玩玩，不要这么卷。🤣

**Reply by tw93** [reply]
🐶

**Comment by 14790897** [comment]
哥，如何理解sam的这句话：‘[2] Interestingly, no one is ever considering going into academia.
’

**Reply by tw93** [reply]
有趣的是，没有人考虑进入学术界。
好比让你读博士研究东西还是找一个好工作，可能更多人会选择后者

**Reply by 14790897** [reply]
我个人的理解：搞学术提升能力的速度不如在初创公司工作
